### PR TITLE
fix: enable filtering for pingprobes

### PR DESCRIPTION
**Pull request by sibicramesh**
Used for retrieving ping probes from a remote namespace using probe ID.

### CODE REVIEW

**Comment by primalmotion** [comment]
This makes the api weird. Why do you need that? The idea was to get the results, not individual probes

File: specs/pingprobe.spec [hunk 1]
```diff
@@ -8,6 +8,10 @@ model:
   description: |-
     Represents the result of a unique ping probe. They are aggregated into a
     PingResult.
+  get:
+    description: Retrieves a ping result.
+    global_parameters:
```
Comment (primalmotion) [inline specs/pingprobe.spec:13]: We don't need filtering here

**Comment by satyamsi** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "get-pp",
    "component": "gaia",
    "pr-id": "515",
    "commit-sha": "dac0ccaa4889dc6234f17a6d61025e379da7066f",
    "pipeline": "master"
  },
  {
    "project": "get-pp",
    "component": "backend",
    "pr-id": "782",
    "commit-sha": "af278a19bd7376d0334b9f4c153370dc0eb68620",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "get-pp",
    "component": "gaia",
    "pr-id": "515",
    "commit-sha": "784e2b28501a85d8f3372300b7aeff9491b5a929",
    "pipeline": "master"
  },
  {
    "project": "get-pp",
    "component": "backend",
    "pr-id": "782",
    "commit-sha": "1af8971bb16018af63749020d448045848ee85dc",
    "pipeline": "master"
  }
]
```

**Pull request merged**